src/styles.css

```diff
@@ -0,0 +1,61 @@
+/* General Styles */
+
+html {
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 10px;
+  background-color: #f5f5f5;
+}
+
+body {
+  margin: 0;
+}
+
+/* Header Styles */
+
+.header {
+  width: 100%;
+  height: 60px;
+  display: flex;
+  text-align: center;
+  align-items: center;
+  background-color: rgb(232, 232, 232);
+  justify-content: space-around;
+  border-bottom: 2px solid black;
+}
+
+/* Body Styles */
+
+.body {
+  display: flex;
+  margin-left: 0px;
+  margin-right: 0px;
+  min-height: calc(100vh - 62px);
+  max-height: 100vh;
+  padding-right: 0px;
+}
+
+.sidebar {
+  display: flex;
+  flex-direction: column;
+  max-width: 30%;
+  width: 400px;
+  background-color: gray;
+  border-right: 1.25px solid black;
+}
+
+.resource {
+  padding: 15px 0px 15px 25px;
+  margin: 0;
+  background-color: lightgray;
+  border-bottom: 1.5px solid black;
+  font-size: 18px;
+  overflow: hidden;
+}
+
+.welcome {
+  min-width: 70%;
+  width: calc(100% - 400px);
+  font-size: 20px;
+  text-align: center;
+  align-self: center;
+}
```

Comment:
Consider splitting header and body styles into separate files and scoping them within the components where they're used.